src/components/OptionEntry.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+
+class OptionEntry extends React.Component {
+    constructor() {
+        super(React.Component);
+    }
+
+    render() {
+        return (
+            <div className="d-flex flex-row justify-content-between">
```

Comment:
All of a sudden, I think this `className` should be a class property because it is inherent to this class. Like this:
```
    className = 'd-flex flex-row justify-content-between';
    render() {
        return (
            <div className={this.className}>
                {this.props.children}
            </div>
        );
    }
```
Although this is basically no more than using one more variable...

Reply:
If you do change it, the same should be done for other classes, too.

Reply:
Agreed